CHANGELOG.md

```diff
@@ -51,6 +51,14 @@ input({
 
 This was added in [pull request #1561: Add `beforeInput(s)` and `beforeInput(s)` options to form groups](https://github.com/nhsuk/nhsuk-frontend/pull/1561).
 
+#### Added a modifier class for text input styles that accept codes and sequences
+
+We've added a new `.nhsuk-input--code` class for text Inputs. This improves readability of text inputs that receive codes and sequecnes (like NHS numbers, security codes or booking references).
```

Comment:
Just got a stray title case `Input` and a little typo

Because "text input(s)" is repeated in the 2nd sentence, shall we keep the link?

```suggestion
We've added a new `.nhsuk-input--code` class for the [text input](https://service-manual.nhs.uk/design-system/components/text-input) component. This improves readability of text inputs that receive codes and sequences (like NHS numbers, security codes or booking references).
```